DaanKort/Competa-dashboard
# Db04 events

## Made the Events you may like section. 

### How to test
Check if its according to design.
Scroll to the sides to see if it works and if the scrollbar is removed
